src/olympia/reviewers/templatetags/jinja_helpers.py

```diff
@@ -221,13 +222,10 @@ def is_expired_lock(context, lock):
 
 
 @library.global_function
-def code_manager_url(path):
-    if not path.startswith('/'):
-        raise ValueError(
-            'Expected a relative path; got: "{}"'.format(path)
-        )
-    # Always return URLs in en-US because the Code Manager is not localized.
-    return '{}/en-US{}'.format(settings.CODE_MANAGER_URL, path)
+def code_manager_url(
+        page, addon_id, version_id, compare_version_id=None):
+    return code_manager_tags.code_manager_url(
+        page, addon_id, version_id, compare_version_id)
```

Comment:
So, no file in this case?

Reply:
Correct. This helper is only used on one page, and that doesn't provide links to specific files, so it doesn't need the `file` feature.